[DOCS] CLI Edits (#7865)
great-expectations/great_expectations: docs/docusaurus/docs/guides/setup/configuring_metadata_stores/how_to_configure_a_validation_result_store_on_a_filesystem.md

### 4. Confirm that the location has been updated by running ``great_expectations store list``

> To make Great Expectations look for Validation Results on the new store, you must set the ``validations_store_name`` variable to the name of your Filesystem Validations Store, which in our example is `shared_validations_filesystem_store`.

Reviewer: ```suggestion
To make Great Expectations look for Validation Results in the new Store, you must set the ``validations_store_name`` variable to the name of your Filesystem Validations Store. In this example, it is `shared_validations_filesystem_store`.
```
